src/main/java/com/example/demo/services/LocationService.java

```diff
@@ -0,0 +1,57 @@
+package com.example.demo.services;
+
+import com.example.demo.entities.LocationEntity;
+import com.example.demo.repositories.LocationEntityRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class LocationService {
+
+    @Autowired private LocationEntityRepository locationEntityRepository;
+
+    public LocationEntity createLocation(LocationEntity location)
+    {
+        return locationEntityRepository.save(location);
+    }
+
+    public List<LocationEntity> findAllLocations()
+    {
+        return (List<LocationEntity>) locationEntityRepository.findAll();
```

Comment:
findAllLocations and findAllLocationsOrdered can be replaced with just the findAllLocationsOrdered method as findAllLocations is just the ordered version with an unpaged and unsorted parameter?

Reply:
Removed, in next story, missed it this one.